src/packets.toit

```diff
@@ -11,6 +11,30 @@ import .topic_filter
 interface PacketIDAck:
   packet_id -> int
 
+class WillConfig:
+  retain/bool
+  qos/int
+  topic/string
+  message/string
```

Comment:
Show this support both strings and byte arrays as the last will message?

Reply:
Good idea! Makes total sense... I'll add support for byte array messages.